docs/data/material/pagesApi.js

```diff
@@ -27,6 +27,7 @@ module.exports = [
   { pathname: '/material-ui/api/checkbox' },
   { pathname: '/material-ui/api/chip' },
   { pathname: '/material-ui/api/circular-progress' },
+  { pathname: '/material-ui/api/click-away-listener' },
```

Comment:
I'm confused about this. Would it mean we will have:

- `@mui/material/ClickAwayListener`
- `@mui/base/ClickAwayListener`
- `@base_ui/react/ClickAwayListener`. I mean, I imagine we need it https://stackoverflow.com/questions/32553158/detect-click-outside-react-component 

<img width="512" alt="SCR-20240713-bcxd" src="https://github.com/user-attachments/assets/bcd23fff-e423-40f7-bc03-e06266fcbe73">